climpred/utils.py

```diff
@@ -331,3 +334,17 @@ def copy_coords_from_to(xro_from, xro_to):
             f'xr.Dataset, found {type(xro_from)} {type(xro_to)}.',
         )
     return xro_to
+
+
+def _load_into_memory(res):
+    """Compute no lazy results."""
+    if dask.is_dask_collection(res):
+        res = res.compute()
+    return res
+
+
+def _transpose_and_rechunk_to(new_chunk_ds, ori_chunk_ds):
```

Comment:
Same as earlier -- clean up docstring and see if circular import is fixed and put into alphabetical order

Reply:
the circular import this is still there, my ealier PR wasnt good enough. lets do this architecture thing at the board in Boulder

Reply:
still recoded in comparisons.py